sentry_sdk/integrations/sanic.py

```diff
@@ -96,14 +96,21 @@ async def sentry_handle_request(self, request, *args, **kwargs):
 
         old_router_get = Router.get
 
-        def sentry_router_get(self, request):
-            # type: (Any, Request) -> Any
-            rv = old_router_get(self, request)
+        def sentry_router_get(self, *args):
+            # type: (Any, Union[Any, Request]) -> Any
+            if version >= (21, 3):
+                rv = old_router_get(self, *args)
+            else:
+                request = args[0]
+                rv = old_router_get(self, request)
```

Comment:
not sure why this branch is necessary, why not always do `*args`?

Reply:
Yeah I agree.. added it initially to make the distinction but I guess no need